reference/UiTPAS.v2.json

```diff
@@ -2615,90 +2709,799 @@
+        "tags": [
+          "Rewards"
+        ],
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "uitpasNumber": {
+                    "type": "string",
+                    "description": "UiTPAS number of the passholder exchanging this reward."
+                  }
+                },
+                "required": [
+                  "uitpasNumber"
+                ]
+              },
+              "examples": {
+                "Example": {
+                  "value": {
+                    "uitpasNumber": "string"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "security": [
+          {
+            "CLIENT_ACCESS_TOKEN": []
+          },
+          {
+            "USER_ACCESS_TOKEN": []
+          }
+        ]
+      },
+      "get": {
+        "summary": "Check reward exchange status for a passholder",
+        "operationId": "get-rewards-id-exchange",
+        "responses": {
+          "202": {
+            "description": "Accepted.\n\nThere is no further content in this response. If the status is Accepted, the given passholder can exchange this reward."
+          },
+          "400": {
+            "description": "Bad Request. Possbile error types:\n\n* https://api.publiq.be/probs/uitpas/social-tariff-expired\n* https://api.publiq.be/probs/uitpas/invalid-uitpas-number\n* https://api.publiq.be/probs/uitpas/reward-period-constraint\n* https://api.publiq.be/probs/uitpas/reward-volume-constraint\n* https://api.publiq.be/probs/uitpas/reward-passholder-constraint\n"
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "description": "Not Found. Possible error types:\n\n* https://api.publiq.be/probs/uitpas/reward-not-found"
+          }
+        },
+        "description": "Check the reward exchange status for a passholder\n\n\nThe caller of this request must have `REWARDS_EXCHANGE` permission for the given organizer.",
+        "security": [
+          {
+            "CLIENT_ACCESS_TOKEN": []
+          },
+          {
+            "USER_ACCESS_TOKEN": []
+          }
+        ],
+        "tags": [
+          "Rewards"
+        ],
+        "parameters": [
+          {
+            "schema": {
+              "type": "string"
+            },
+            "in": "query",
+            "required": true,
+            "name": "uitpasNumber",
+            "description": "UiTPAS number identifying the passholder who wants to exchange this reward."
+          }
+        ]
+      }
+    },
+    "/rewards": {
+      "post": {
+        "summary": "Create new reward",
+        "operationId": "post-rewards",
+        "responses": {
```

Comment:
Is this not missing a 400 response? For when the creation fails due to bad data etc?